vm/src/stdlib/syslog.rs

```diff
@@ -0,0 +1,106 @@
+pub(crate) use syslog::make_module;
+
+#[pymodule(name = "syslog")]
+mod syslog {
+    use crate::builtins::pystr::{PyStr, PyStrRef};
+    use crate::function::{OptionalArg, OptionalOption};
+    use crate::pyobject::PyValue;
+    use crate::utils::ToCString;
+    use crate::vm::VirtualMachine;
+    use crate::{PyObjectRef, PyResult, TryFromObject};
+
+    #[pyattr]
+    use libc::{
+        LOG_ALERT, LOG_AUTH, LOG_CONS, LOG_CRIT, LOG_DAEMON, LOG_DEBUG, LOG_EMERG, LOG_ERR,
+        LOG_INFO, LOG_KERN, LOG_LOCAL0, LOG_LOCAL1, LOG_LOCAL2, LOG_LOCAL3, LOG_LOCAL4, LOG_LOCAL5,
+        LOG_LOCAL6, LOG_LOCAL7, LOG_LPR, LOG_MAIL, LOG_NDELAY, LOG_NEWS, LOG_NOTICE, LOG_NOWAIT,
+        LOG_ODELAY, LOG_PID, LOG_SYSLOG, LOG_USER, LOG_UUCP, LOG_WARNING,
+    };
+
+    #[cfg(not(target_os = "redox"))]
+    #[pyattr]
+    use libc::{LOG_AUTHPRIV, LOG_CRON, LOG_PERROR};
+
+    fn get_argv(vm: &VirtualMachine) -> Option<PyStrRef> {
+        if let Some(argv) = vm.state.settings.argv.first() {
+            if !argv.is_empty() {
+                return Some(
+                    PyStr::from(match argv.find('\\') {
+                        Some(value) => argv[value..].to_owned(),
+                        None => argv.to_owned(),
```

Comment:
```suggestion
                        Some(value) => argv[value..].to_owned(),
                        None => argv.to_owned(),
```

PyStr::from works for `&str` too